src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c

```diff
@@ -1783,18 +1783,18 @@ int xocl_userpf_probe(struct pci_dev *pdev,
 
 	xocl_drvinst_set_offline(xdev, false);
 
-	if (!pci_set_dma_mask(pdev, DMA_BIT_MASK(64))) {
```

Comment:
It is unclear why all these changes are no longer dependent of the kernel version.

Reply:
Good question :) In my above comment I wrote that `pci_dma_*` functions are wrappers around `dma_*` functions. `dma_*` functions probably existed for a long time. Given that the patches to remove `pci_dma_*` functions are fairly trivial, I do not think guarding is required.

The question is "What was the contribution of `pci-dma-compat.h`?` I understand that [this is a legacy API](https://lore.kernel.org/kernel-janitors/20200421081257.GA131897@infradead.org/) that can be easily replaced.

[This post](https://lore.kernel.org/kernel-janitors/20200716192821.321233-1-christophe.jaillet@wanadoo.fr/) mentions that some constants (e.g., `GFP_*`) should be manually replaced, but I did not encounter these in XRT.